docs/build/sdk/v2/selling-assets.md

```diff
@@ -0,0 +1,53 @@
+# Selling Assets
```

Comment:
It's a bit strange that _selling assets_ gets its own chapter, but _buying assets_ doesn't. Symmetry and all... ;)

Reply:
Yeah so the `Making A Prediction` category is also a chapter, but if I also put it as a sub item the next button is recked. But I can rename it to `Buying Assets`?